decidim-core/app/cells/decidim/redesigned_profile/details.erb

```diff
@@ -0,0 +1,12 @@
+<div class="profile__details">
+  <h1 class="h3"><%= presented_profile.name %></h1>
```

Comment:
We have a problem with the profile page H1 elements right now as all of the profile pages have exactly the same H1, although the content differs for all pages.

Here is some further reading on the topic:
https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

Related WCAG 2.1 criteria:
-  [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)
-  [2.4.6 Headings and Labels](https://www.w3.org/TR/WCAG21/#headings-and-labels)

Reply:
Moved to #9670.